AliveSFU/AliveSFU/ExerciseTileViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ExerciseTileViewController.swift
+//  AliveSFU
+//
+//  Created by Liam O'Shaughnessy on 2016-10-28.
+//  Copyright © 2016 SimonDevs. All rights reserved.
+//
+
+import UIKit
+
+class ExerciseTileViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor.black.withAlphaComponent(0.8)
+        self.showAnimate()
+        
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func closePopUp(_ sender: Any) {
```

Comment:
This will break now because we're closing the popover a different way. Either implement the new way or you can just leave a comment that this is broken and we can fix it later :)